assets/src/utils/charts.ts

```diff
@@ -44,3 +51,12 @@ export function createCharts(
     container
   );
 }
+
+function getChartTitle(metric: string): string {
```

Comment:
if you change the `Map` to object then just use ternary `metricToTitle[metric] ?? metric`